app/views/rules/_rule.html.erb

```diff
@@ -1,44 +1,62 @@
 <%# locals: (rule:) %>
-
-<div class="flex justify-between items-center gap-4 bg-white shadow-border-xs rounded-md p-4">
+<div class="flex justify-between items-center p-4 <%= 'text-secondary' unless rule.active? %>">
   <div class="text-sm space-y-1.5">
+    <% if rule.name.present? %>
+      <h3 class="font-medium text-md text-primary"><%= rule.name %></h3>
+    <% end %>
+
     <% if rule.conditions.any? %>
-      <p class="flex items-center flex-wrap gap-1.5">
-        <span class="px-2 py-1 border border-alpha-black-200 rounded-full">
-          <% if rule.conditions.first.compound? %>
-            If <%= rule.conditions.first.sub_conditions.first.filter.label %> <%= rule.conditions.first.sub_conditions.first.operator %> <%= rule.conditions.first.sub_conditions.first.value_display %>
+      <div class="flex items-center gap-2 mt-1">
+        <div class="flex items-center gap-1 text-secondary w-16 shrink-0">
+          <span class="font-mono text-xs">IF</span>
+        </div>
+        <p class="flex items-center flex-wrap gap-1.5 m-0">
+          <span class="px-2 py-1 border border-secondary rounded-full">
+            <% if rule.conditions.first.compound? %>
+              <%= rule.conditions.first.sub_conditions.first.filter.label %> <%= rule.conditions.first.sub_conditions.first.operator %> <%= rule.conditions.first.sub_conditions.first.value_display %>
+            <% else %>
+              <%= rule.conditions.first.filter.label %> <%= rule.conditions.first.operator %> <%= rule.conditions.first.value_display %>
+            <% end %>
+          </span>
+          <% if rule.conditions.count > 1 %>
+            and <%= rule.conditions.count - 1 %> more <%= rule.conditions.count - 1 == 1 ? "condition" : "conditions" %>
+          <% end %>
+        </p>
```

Comment:
I had thrown a lot of this checking in a bit hastily for a quick fix a few weeks back and I'm totally fine keeping it as is for now.  But probably worth noting that we'll eventually need to capture this a bit more clearly.  Just an idea, but something that looks more like:

```erb
<span class="px-2 py-1 border border-secondary rounded-full">
  <%= rule.primary_condition_title %>
</span>

<% if rule.conditions.count > 1 %>
  and <%= rule.conditions.count - 1 %> more <%= rule.conditions.count - 1 == 1 ? "condition" : "conditions" %>
<% end %>
```

Reply:
Agreed! I have seen some other weird stuff on this condition thing too so I threw this comment and my other experience into https://github.com/maybe-finance/maybe/issues/2230